core/tests/test_models.py

```diff
@@ -131,3 +132,42 @@ def test_tags_removed_on_post_update(user: User, community: Community) -> None:
     assert "tag3" in tag_names
     assert "tag1" not in tag_names
     assert "tag2" not in tag_names
+
+
+@pytest.mark.django_db()
+def test_create_community_user(user: User, community: Community) -> None:
+    community_user: CommunityMember = CommunityMember.objects.create(
+        user=user,
+        community=community,
+        role=CommunityMember.ADMIN,
+    )
+
+    assert community_user.user == user

```

Comment:
Hmm
this test would be much better if asserts were from community not from community_user (also mind name).
Actually this test does not make sense as it only checks if entry in db was created, and not even refreshing it. 
So the proper way would be (pseudocode)
assert community.users.filter(user = user) == user